transforms/__testfixtures__/update-react-imports/react-not-removed.output.js

```diff
@@ -0,0 +1,7 @@
+import * as React from "react";
+
+React.createElement('div', {});
```

Comment:
feels like the destructuring would still be useful here. Is the decision to bail completely?

Reply:
Yeah to simplify the code, we don't destructure if either React is already being used or if the variable we're destructuring to (ex. `createElement`) is already being used. Open to changing though.

Reply:
I don't think the plan is to ever completely remove `React.createElement`. It'll still be there for people who want to manually create elements. We do plan to eventually stop letting the JSX transform use t.